
ked mam parameter par vo funkcii kt. volam
je deklarovanu premennu par, tak vyhlasi chybu  (neviem ci je aktualne
========================================s
> function foo() as integer
> a as integer
> a = 2
> return a
> end function
> scope
> foo()
> end scope
-hodi semanticku chybu
=========================================
volanie funkcie je pokazene
> function foo(par as integer) as integer
> end function
> scope
> foo(3)
> end scope
len vytvori frame, nezavola funkciu
=========================================
return vo funckii
    -neskonci funkci ked je return
    -ked je prazdny return tak popuje bez predchadzajuceho pushu
function foo(par as integer, par2 as integer) as integer
dim a as double
return 5
a = 2.3
return a
end function
scope
dim a as integer = 5
foo(3,a)
end scope
=========================================
prazdny subor vyhadzyje syn err
